e2e/Utils.js

```diff
@@ -225,3 +225,8 @@ export const normalizeIcons = (str: string) => {
     .replace(new RegExp('\u00D7', 'g'), '\u2715')
     .replace(new RegExp('\u221A', 'g'), '\u2713');
 };
+
+export const countExecutedTests = (str: string): number => {
```

Comment:
Did you take a look at the `json` function in the `runJest` e2e test util file? That might be an easier way to count the number of tests run (we do  it in many e2e tests already) unless I'm missing a reason why it needs to be done this way here :)

Reply:
That's a good point! Then we can assert the number of skipped tests as well